src/Peripherals/InternetPeripheral.cpp

```diff
@@ -23,8 +24,9 @@ bool InternetPeripheral::attemptConnection(QString ipAddress, unsigned short por
     {
         channel_ = AmqpClient::Channel::Create(ipAddress.toStdString(), port);
     }
-    catch (AmqpClient::AmqpException::exception&)
+    catch (AmqpClient::AmqpException::exception& exception)
     {
+        qDebug() << exception.what();
```

Comment:
Can you add a TODO to have this debug message filter up to the UI. Otherwise looks good.